asn1ate/asn2quickder.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/python
+#
+# asn2quickder -- Generate header files for C for use with Quick `n' Easy DER
+#
+# This program owes a lot to asn1ate, which was built to generate pyasn1
+# classes, but which was so well-written that it could be extended with a
+# code generator for Quick DER.
+#
+# Much of the code below is diagonally inspired on the pyasn1 backend, so
+# a very big thank you to Schneider Electric Buildings AB for helping to
+# make this program possible!
+#
+# From: Rick van Rein <rick@openfortress.nl>
+
+
+import sys
+import os.path
+
+from asn1ate import parser
+from asn1ate.sema import * 
+
+
+def toCsym (name):
```

Comment:
I've called this `sanitize_identifier` in the Python generator module, maybe that would make sense here as well? I'm not too happy about the mixed-case name :-)